packages/opentelemetry-plugin-postgres/opentelemetry-plugin-pg-pool/test/pg-pool.test.ts

```diff
@@ -94,8 +94,8 @@ describe('pg-pool@2.x', () => {
   let pool: pgPool<pg.Client>;
   const registry = new NodeTracerRegistry();
   const logger = new NoopLogger();
-  const testPostgres = process.env.TEST_POSTGRES; // For CI: assumes local postgres db is already available
-  const testPostgresLocally = process.env.TEST_POSTGRES_LOCAL; // For local: spins up local postgres db via docker
+  const testPostgres = process.env.RUN_POSTGRES_TESTS; // For CI: assumes local postgres db is already available
```

Comment:
Does this mean tests have not been running?

Reply:
I'm afraid so, but it should been running before test-utils package.